lib/workers/repository/init/cache.ts

```diff
@@ -1,21 +1,18 @@
 import fs from 'fs-extra';
 import * as npmApi from '../../../modules/datasource/npm';
-import * as memCache from '../../../util/cache/memory';
 import * as repositoryCache from '../../../util/cache/repository';
 import { initRepoCache } from '../../../util/cache/repository/init';
 import { privateCacheDir } from '../../../util/fs';
 import type { WorkerPlatformConfig } from './apis';
 
 export async function resetCaches(): Promise<void> {
-  memCache.reset();
```

Comment:
Why is reset no longer needed?

Reply:
Because `resetCaches()` is called just before `memCache.init()`, so it doesn't make much sense